compiler/builtins/bigint.ts

```diff
@@ -0,0 +1,100 @@
+import type {} from './porffor.d.ts';
+
+export const BigInt = function (value: number): any {
+  let result: BigInt = Porffor.wasm`
+    local.get ${value}

```

Comment:
Doesn't this need to cast the argument to a number first?

Reply:
Yes, again, it's incomplete currently.